applications/FSIApplication/python_scripts/partitioned_fsi_base_solver.py

```diff
@@ -109,14 +97,14 @@ def __init__(self, model, project_parameters):
         if mesh_solver_settings.Has("calculate_mesh_velocity"):
             if not mesh_solver_settings["calculate_mesh_velocity"].GetBool():
                 mesh_solver_settings.SetValue("calculate_mesh_velocity", True)
-                self._PrintWarningOnRankZero("","Mesh velocity calculation was desactivated. Switching \"calculate_mesh_velocity\" on")
+                KratosMultiphysics.Logger.PrintWarning("","Mesh velocity calculation was desactivated. Switching \"calculate_mesh_velocity\" on")
```

Comment:
```suggestion
                KratosMultiphysics.Logger.PrintWarning("","Mesh velocity calculation was deactivated. Switching \"calculate_mesh_velocity\" on")
```

Reply:
I missed that one. Thanks for spotting it!